build.sbt

```diff
@@ -109,7 +109,8 @@ lazy val activityStreamImpl = project("activity-stream-impl")
     resolvers += bintrayRepo("hseeberger", "maven"),
     libraryDependencies ++= Seq(
       lagomJavadslCluster,
-      lagomJavadslTestKit
+      lagomJavadslTestKit,
+      "com.test" %% "test" % "1.0"
```

Comment:
Is this an intentional inclusion?

Reply:
It wasn't :man_facepalming:  -- will be removed.